processing.js

```diff
@@ -2497,7 +2497,7 @@ module.exports = function(options, undef) {
      * @param {float} sy  the amount to scale on the y-axis
      */
     scale: function(sx, sy) {
-      if (sx && !sy) {
+      if (sx && sy === undef) {
```

Comment:
Looks like it's because of this commit from the 1st of Feb 2016:
https://github.com/processing-js/processing-js/commit/991773de8d83b4d751b0c0a051bb151e4f175abc

processing.js hasn't been updated since the 18th Sept 2015 though. So auto-generating processing.js will introduce these changes. 

If you consider these changes bad they'd have to be fixed in src/Objects/PMatrix2D.js and src/Objects/PMatrix3D.js.